<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>consult</class>
 <widget class="QMainWindow" name="consult">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Consultar Exame</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>551</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QLabel" name="label_search">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btn_consult">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string>Pesquisar</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLineEdit" name="edit_cpf">
       <property name="minimumSize">
        <size>
         <width>194</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>194</width>
         <height>28</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>Insira o CPF do Cliente</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="4">
      <widget class="QTableWidget" name="table_consult">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="dragDropOverwriteMode">
        <bool>false</bool>
       </property>
       <property name="gridStyle">
        <enum>Qt::DashLine</enum>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>250</number>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>150</number>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>30</number>
       </attribute>
       <attribute name="verticalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Nome Paciente</string>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Data do Exame</string>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>581</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #6a9b4a;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>575</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_consult</sender>
   <signal>clicked()</signal>
   <receiver>consult</receiver>
   <slot>on_btnSearch_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>table_consult</sender>
   <signal>cellDoubleClicked(int,int)</signal>
   <receiver>consult</receiver>
   <slot>on_modelIndex_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>470</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>504</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_btnSearch_clicked()</slot>
  <slot>on_modelIndex_clicked()</slot>
 </slots>
</ui>
